oxidation/parsec_cli/parsec_sequestre_dump/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "parsec_cli"
+version = "0.0.0"
+edition = "2021"
+license = " BUSL-1.1"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+parsec_api_crypto = { path = "../../crates/parsec_api_crypto" }
+
+diesel = { version = "2.0.0-rc.0", features = ["sqlite"], git = "https://github.com/diesel-rs/diesel" }
+clap = { version = "3.1.18", features = ["derive"] }
+rmp-serde = "1.0.0"
```

Comment:
I don't think `serde`&`rmp-serde`  are needed since we won't create our own schemas (so we need parsec_api_types` instead)